docs/storybook/src/frontstage/SplitViewport.stories.tsx

```diff
@@ -0,0 +1,114 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import {
+  ConditionalStringValue,
+  StandardContentLayouts,
+} from "@itwin/appui-abstract";
+import {
+  ContentProps,
+  IModelViewportControl,
+  SyncUiEventId,
+  ToolbarItemUtilities,
+  UiFramework,
+} from "@itwin/appui-react";
+import { AppUiDecorator } from "../AppUiDecorator";
+import { Page } from "../AppUiStory";
+import { createFrontstageProvider, removeProperty } from "../Utils";
+import { SplitViewportStory } from "./SplitViewport";
+import {
+  Svg2D,
+  Svg3D,
+  SvgWindow,
+  SvgWindowSplitVertical,
+} from "@itwin/itwinui-icons-react";
+import { ConditionalIconItem } from "@itwin/core-react";
+
+const meta = {
+  title: "Frontstage/SplitViewport",
+  component: SplitViewportStory,
+  tags: ["autodocs"],
+  decorators: [AppUiDecorator],
+  parameters: {
+    docs: {
+      page: () => <Page />,
+    },
+    layout: "fullscreen",
+  },
+  argTypes: {
+    frontstageProviders: removeProperty(),
+    itemProviders: removeProperty(),
+  },
+} satisfies Meta<typeof SplitViewportStory>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+const contentPropsArray: ContentProps[] = [];
+contentPropsArray.push({
+  id: "imodel-view-0",
+  classId: IModelViewportControl.id,
+});
+contentPropsArray.push({
+  id: "imodel-view-1",
+  classId: IModelViewportControl.id,
+});
+
+let leftViewportActive = false;
+const testIcon1 = () => {
+  return leftViewportActive ? <SvgWindowSplitVertical /> : <SvgWindow />;
+};
+const testIcon2 = () => {
+  return leftViewportActive ? <Svg2D /> : <Svg3D />;
+};
+
+UiFramework.content.onActiveContentChangedEvent.addListener(() => {
+  leftViewportActive = !leftViewportActive;
+});
+
+export const Default: Story = {
+  args: {
+    frontstageProviders: [
+      createFrontstageProvider({
+        contentGroupProps: {
+          id: "split-vertical-group",
+          layout: StandardContentLayouts.twoVerticalSplit,
+          contents: contentPropsArray,
+        },
+      }),
+    ],
+    itemProviders: [
+      {
+        id: "toolbar",
+        provideToolbarItems: () => [
+          ToolbarItemUtilities.createActionItem(
+            `Test1and2`,
+            0,
+            new ConditionalIconItem(testIcon1, [
+              SyncUiEventId.ActiveContentChanged,
+            ]),
+            new ConditionalStringValue(
+              () => (leftViewportActive ? "Test 1" : "Test 2"),
+              [SyncUiEventId.ActiveContentChanged]
+            ),
+            () => undefined
+          ),
+          ToolbarItemUtilities.createActionItem(
+            `Test3and4`,
+            0,
+            new ConditionalIconItem(testIcon2, [
+              SyncUiEventId.ActiveContentChanged,
+            ]),
+            new ConditionalStringValue(
+              () => (leftViewportActive ? "Test 3" : "Test 4"),
+              [SyncUiEventId.ActiveContentChanged]
+            ),
+            () => undefined
+          ),
+        ],
```

Comment:
Can you add a 3rd items that show the `isHidden` (which you can set with the `overrides` param of `createActionItem`) and add a `isDisabled` to the second one, so the example showcase even more info ?